frontend/src/app/store/reducers/cases.ts

```diff
@@ -445,4 +468,69 @@ const parseAssessmentResponse = async (res: CaseFileDto, officers: Officer[]): P
   }
 }
 
+export const createReview = (complaintId: string, isReviewRequired: boolean, reviewComplete: any ): AppThunk => async (dispatch, getState) => {
+  const {
+    app: { profile },
+    cases: { caseId }
+  } = getState();
+  let reviewInput = {
+    reviewInput: {
```

Comment:
I feel like there's a missing property here for the actor when you're creating and updating your review.

Reply:
`actor` is a property inside the reviewComplete tho

Reply:
Marking a file as "Requires Review" isn't action (at least not yet) as there is no place to record the actor / date.   It could be inferred, but that would be a future enhancement.

Reply:
For "Review require", there's no action added to `action` table, so there is no actor, just update the checkbox in `review_required_ind` field in `case_file` table.
For "Review complete", it's an action hence it will be added in `action` table, with the actor defined in file `hwcr-file-review.tsx` line 57